packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -127,6 +127,11 @@ class MenuContent extends React.Component {
     return { isToggleableMenu: this.isToggleable(), shouldReserveSpaceForIcon: this.shouldReserveSpaceForIcon() };
   }
 
+  componentDidMount() {
+    // Set focus to first focusable menu item
+    this.contentNode.querySelectorAll('li[class*="item"][data-menuitem="true"][tabindex="0"]')[0].focus();
```

Comment:
```suggestion
    const items = this.contentNode.querySelectorAll('[data-terra-menu-interactive-item="true"]);
    
   if (items.length) {
       items[0].focus();
   }
```
if query selector returns zero elements then focus() method will throw error without condition check.